frontend/src/pages/connectionTypes/manage/ManageConnectionTypePage.tsx

```diff
@@ -139,7 +150,20 @@ const ManageConnectionTypePage: React.FC<Props> = ({ prefill, isEdit, onSave })
                 setData={setConnectionNameDesc}
                 autoFocusName
               />
-              <FormGroup label="Enable">
+              <FormGroup label="Category" fieldId="connection-type-category" isRequired>
+                <MultiSelection
+                  inputId="connection-type-category"
+                  toggleTestId="connection-type-category"
```

Comment:
We should be able to add a test id to the `MultiSelection` itself separate from the toggle button. This will allow you to relatively find the menu items instead of searching globally. 

```suggestion
                  data-testid="connection-type-category"
                  toggleTestId="connection-type-category-toggle"
```